src/group_theory/nilpotent.lean

```diff
@@ -577,6 +577,47 @@ lemma nilpotency_class_quotient_le (H : subgroup G) [H.normal] [h : is_nilpotent
 lemma derived_le_lower_central (n : ℕ) : derived_series G n ≤ lower_central_series G n :=
 by { induction n with i ih, { simp }, { apply general_commutator_mono ih, simp } }
 
+lemma _root_.comm_group.center_eq_top {G : Type*} [comm_group G] : center G = ⊤ := begin
```

Comment:
This indeed seems to be missing for subgroups:
```
src/algebra/algebra/subalgebra.lean
989:@[simp] lemma center_eq_top (A : Type*) [comm_semiring A] [algebra R A] : center R A = ⊤ :=

src/algebra/lie/semisimple.lean
90:  rw [is_lie_abelian_iff_center_eq_top R L, center_eq_bot_of_semisimple] at h,

src/algebra/lie/abelian.lean
230:lemma is_lie_abelian_iff_center_eq_top : is_lie_abelian L ↔ center R L = ⊤ :=

src/ring_theory/subsemiring/basic.lean
410:@[simp] lemma center_eq_top (R) [comm_semiring R] : center R = ⊤ :=

src/group_theory/submonoid/center.lean
149:@[simp] lemma center_eq_top : center M = ⊤ :=

src/ring_theory/subring/basic.lean
526:@[simp] lemma center_eq_top (R) [comm_ring R] : center R = ⊤ :=
```
Would you please move it to just after the definition of `center`?